src/test/java/shop/RealItemTest.java

```diff
@@ -0,0 +1,35 @@
+package shop;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RealItemTest {
+    RealItem realItem;
+    final double itemWeight = 55.7;
+
+    @org.junit.jupiter.api.BeforeEach
+    void setUp() {
+        realItem = new RealItem();
+        realItem.setWeight(itemWeight);
+    }
+
+    @org.junit.jupiter.api.AfterEach
+    void tearDown() {
+        realItem = null;
+    }
+
+    @Test
+    void getWeight() {
+        assertEquals(realItem.getWeight(), itemWeight);
```

Comment:
Actually RealItem class was designed as class without any business logic and it is hard to develop unit tests for it. Even I can say that it doesn't make sense. The only test which can be developed for overriden toString() method. Please remove test for setter/getter, because it is simple operation